server/leafnode.go

```diff
@@ -2087,7 +2086,7 @@ func (acc *Account) updateLeafNodes(sub *subscription, delta int32) {
 	defer acc.lmu.RUnlock()
 
 	// Do this once.
-	subject := string(sub.subject)
+	subject := bytesToString(sub.subject)
```

Comment:
No, because again this is passed to `ln.canSubscribe()` few lines down in the `for` loop.